saturn-r/.env_deps

```diff
@@ -1,3 +1,3 @@
-VERSION=2021.04.19-1
-SATURNBASE_IMAGE=saturncloud/saturnbase:2021.02.22
-IMAGE=saturncloud/saturn-r:2021.04.19-1
+VERSION=2021.07.26
+SATURNBASE_IMAGE=saturncloud/saturnbase:2021.07.26
```

Comment:
@forana should this be `SATURNBASE_IMAGE=saturncloud/saturnbase:2021.07.26-1` ? I wasn't sure about the difference

Reply:
nope, those are (currently) both pointers to the same thing. `release-images` will overwrite this, anyway.